lib/rack_middleware/enforce_ssl.rb

```diff
@@ -0,0 +1,21 @@
+class EnforceSsl
```

Comment:
Did you consider using the default Rails `force_ssl`? http://guides.rubyonrails.org/configuring.html#rails-general-configuration Why did you end up writing own implementation?